### ISSUE TITLE
This Project is exculded from Hacktoberfest2023

### ISSUE THREAD

**Issue by GD0910** [nikohoffren/fork-commit-merge#965]

**Comment by nikohoffren** [comment]
Yes, i'm aware of this and that's why this project does not have `hacktoberfest` or `hacktoberfest-accepted` labels. There is some information about the situation in the Tasks section of [README](https://github.com/nikohoffren/fork-commit-merge#tasks).
